File: cfgs/java_cfgs_foo/COUNT_DISTINCT_NON_NEGATIVE_PAIRS_X_Y_SATISFY_INEQUALITY_XX_YY_N_2.java
Nodes
1;special;;;
2;statement;int res = 0;14;
3;statement;int x = 0;15;
4;conditional;(foo_1 * foo_2) < foo_3;16;
5;statement;int y = 0;17;
6;conditional;((foo_1 * foo_2) + (foo_3 * foo_4)) < foo_5;18;
7;statement;res++;19;
8;statement;y++;19;
9;statement;x++;17;
10;statement;return foo_1;20;
11;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;10;
4;6;
5;7;9;
6;8;
7;6;
8;4;
9;11;
10;

File: cfgs/pythoncfgs/COUNT_DISTINCT_NON_NEGATIVE_PAIRS_X_Y_SATISFY_INEQUALITY_XX_YY_N_2.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = 0;9;
3;conditional;foo_1 * foo_2 < foo_3;10;
4;statement;return foo_1;16;
5;statement;foo_1 = 0;11;
6;conditional;foo_1 * foo_2 + foo_3 * foo_4 < foo_5;12;
7;statement;foo_1 = foo_2 + 1;15;
8;statement;foo_1 = foo_2 + 1;13;
9;statement;foo_1 = foo_2 + 1;14;
10;exit;;
Edges
0;2;
1;3;
2;4;5;
3;10;
4;6;
5;7;8;
6;3;
7;9;
8;6;
9;
